src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -46,6 +46,18 @@ public new abstract class Arm64 : ArmBase.Arm64
             /// </summary>
             public static Vector128<double> Add(Vector128<double> left, Vector128<double> right) { throw new PlatformNotSupportedException(); }
 
+            // /// <summary>
+            // /// float64x1_t vbsl_f64 (uint64x1_t a, float64x1_t b, float64x1_t c)
+            // ///   A64: BSL Vd, Vn, Vm
+            // /// </summary>
+            // public static Vector64<double> BitwiseSelect(Vector64<ulong> select, Vector64<double> left, Vector64<double> right) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// float64x2_t vbslq_f64 (uint64x2_t a, float64x2_t b, float64x2_t c)
+            ///   A64: BSL Vd, Vn, Vm
+            /// </summary>
+            public static Vector128<double> BitwiseSelect(Vector128<ulong> select, Vector128<double> left, Vector128<double> right) { throw new PlatformNotSupportedException(); }
```

Comment:
Why are these Arm64 only? both `BSL` and `VBSL` treat this as a sequence of bits, and so the underlying type doesn't matter.

Reply:
Just to be clear - your question is with regard to the eventual support on Arm32, not that we expect to immediately enable intrinsics on Arm32, correct?

Reply:
Correct. If they are eventually enabled on ARM32, this API would be implementable and so should live under `AdvSimd` rather than `AdvSimd.Arm64`.

Reply:
Fixed